src/main/java/org/spongepowered/common/event/tracking/phase/plugin/PluginPhaseState.java

```diff
@@ -29,7 +29,7 @@
 import org.spongepowered.common.event.tracking.phase.TrackingPhase;
 import org.spongepowered.common.event.tracking.phase.TrackingPhases;
 
-public abstract class PluginPhaseState<P extends PluginPhaseContext<P>> implements IPhaseState<P> {
+public abstract class PluginPhaseState<P extends PhaseContext<P>> implements IPhaseState<P> {
```

Comment:
Why?

Reply:
This was to allow `CustomExplosionState` to use the same context class as `ExplosionState` (of `GeneralPhase`); there's pretty much no difference between the two types.